spaceship.zsh

```diff
@@ -274,6 +275,13 @@ SPACESHIP_EXIT_CODE_SUFFIX="${SPACESHIP_EXIT_CODE_SUFFIX:=" "}"
 SPACESHIP_EXIT_CODE_SYMBOl="${SPACESHIP_EXIT_CODE_SYMBOl:="✘"}"
 SPACESHIP_EXIT_CODE_COLOR="${SPACESHIP_EXIT_CODE_COLOR:="red"}"
 
+# AWS
+SPACESHIP_AWS_SHOW="${SPACESHIP_AWS_SHOW:=true}"
+SPACESHIP_AWS_PREFIX="${SPACESHIP_AWS_PREFIX:="using "}"
```

Comment:
You were probably referring to line 808 which had `on` instead of `using`. Its fixed now :)